br/pkg/backup/client.go

```diff
@@ -237,9 +237,9 @@ func (bc *Client) SetStorageAndCheckNotInUse(
 			"please specify a correct backup directory!", bc.storage.URI()+"/"+metautil.MetaFile)
 	}
 	// use checkpoint mode if checkpoint meta exists
-	exist, err = bc.storage.FileExists(ctx, checkpoint.CheckpointMetaPath)
```

Comment:
Move the `/checkpoint.meta` to `/checkpoints/backup/checkpoint.meta`.
Move the `/checkpoints/checkpoint.lock` to `/checkpoints/backup/checkpoint.lock`.
Move the  `/checkpoints/data/` to `/checkpoints/backup/data/`.
Move the  `/checkpoints/checksum/` to `/checkpoints/backup/checksum/`.